### ISSUE TITLE
AUR package?

### ISSUE THREAD

**Issue by catemiko** [FaridZelli/ISW-Modern#12]
Would there an aur version of that utility?

**Comment by FaridZelli** [comment]
No, but I'm planning to simplify the installation process and switch to msi-ec.
   
It'll be a single script which can install it on most distros, you'll no longer have to go through multiple steps. Arch, Fedora, openSUSE, Debian/Ubuntu, and Solus can all be handled by the same script minus variations in dependencies. Systemd-less distros could require additional steps, so that'll be something to improve on in the future. Same goes for immutable distros such as Fedora Silverblue/Kinoite and openSUSE MicroOS.
